shared/kafka-connect/compose.yml

```diff
@@ -8,6 +8,7 @@ services:
     ports:
       - "8083:8083"
     environment:
+      KAFKA_HEAP_OPTS: "-Xms10G -Xmx10G"
```

Comment:
Shouldn't this be `CONNECT_HEAP_OPTS`? 🤔

Reply:
I need to document this, but for whatever reason, CONNECT_HEAP_OPTS didn't actually have an effect. It's only when I set KAFKA_HEAP_OPTS that I saw the Kafka Connect container actually consuming more memory (went up to 7.something GB)